__tests__/spec/6_validate_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../lib/config.rb'
+require_relative '../lib/voucherify_data.rb'
+require_relative '../lib/stackables.rb'
```

Comment:
To stackables może tak zostać. Jednak w środku może zmienić nazwy paramertów metod

`def validate_stacked_discounts(stackable_discounts_api_instance, voucher_id)` na 
`def validate_stacked_discounts(valications_api_instance, voucher_id)` i tak samo w `redeem_stacked_discounts`